integration/pull_test.go

```diff
@@ -146,14 +147,14 @@ func TestOptimizeConsistentSociArtifact(t *testing.T) {
 }
 
 func TestLazyPullWithSparseIndex(t *testing.T) {
+	imageName := rabbitmqImage
+
 	regConfig := newRegistryConfig()
 
 	sh, done := newShellWithRegistry(t, regConfig)
 	defer done()
 
-	const imageName = "rethinkdb@sha256:4452aadba3e99771ff3559735dab16279c5a352359d79f38737c6fdca941c6e5"
-	const imageManifestDigest = "sha256:4452aadba3e99771ff3559735dab16279c5a352359d79f38737c6fdca941c6e5"
-	const minLayerSize = 10000000
+	_, minLayerSize, _ := middleSizeLayerInfo(t, sh, dockerhub(imageName))
```

Comment:
nit: might be safer to move this after line 159 `rebootContainerd` to ensure the test itself provides running containerd/soci